airbyte-integrations/connectors/source-woocommerce/pyproject.toml

```diff
@@ -17,7 +17,7 @@ include = "source_woocommerce"
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-airbyte-cdk = "1.0.0"
+airbyte-cdk = "3.8"
```

Comment:
why 3.8? Latest version is 4.0.2

Reply:
I wanted to use 4, but since 3.9, I got `AirbyteTracedException('generator raised StopIteration')` errors. Would it be acceptable to merge this with this relatively recent version and create a task for updating to v4? The reason is this may take considerable time debugging, which is not the main focus of this task, which I want to merge as soon as possible.

Reply:
that sounds like a problem that should be investigated